File: src/Example3Java/EX03.java
package Example3Java;

import java.util.Scanner;

public class EX03 {
    public static void main(String[] args) {
        System.out.println("Enter number : ");
        String number = new Scanner(System.in).nextLine();
        String str = number.substring(0, number.indexOf("."));
        String str2 = number.substring(number.indexOf("."), number.indexOf(".") + 3);
        System.out.println("number : " + str + str2);
    }
}

File: src/Example3Kotlin/EX03.kt
import java.util.*

object EX03 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter number: ")
        val number = Scanner(System.`in`).nextLine()
        val str = number.substring(0, number.indexOf("."))
        val str2 = number.substring(number.indexOf("."), number.indexOf(".") + 3)
        println("number : $str$str2")
    }
}
